eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <PackageReference Include="Wix" Version="3.11.1" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
@pranavkm am I correct changes to this file can be reverted❔ I asked something similar in your P8 PR.

Reply:
It seems to be required to compile now that this project no longer references the analyzer.